### PR TITLE
contracts

### PR DESCRIPTION
**Pull request by mmusin**

### CODE REVIEW

File: contract/Lykke.Service.Assets.Contract/Events/AssetUpdatedEvent.cs [hunk 1]
```diff
@@ -0,0 +1,40 @@
+﻿using MessagePack;
+
+namespace Lykke.Service.Assets.Contract.Events
+{
+    [MessagePackObject(keyAsPropertyName: true)]
+    public class AssetUpdatedEvent
+    {
```
Comment (polonzer) [inline contract/Lykke.Service.Assets.Contract/Events/AssetUpdatedEvent.cs]: Why was this set of fields chosen?

File: contract/Lykke.Service.Assets.Contract/Events/AssetCreatedEvent.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
+﻿using MessagePack;
+
+namespace Lykke.Service.Assets.Contract.Events
+{
+    [MessagePackObject(keyAsPropertyName: true)]
+    public class AssetCreatedEvent
+    {
```
Comment (polonzer) [inline contract/Lykke.Service.Assets.Contract/Events/AssetCreatedEvent.cs]: Why was this set of fields chosen?

File: contract/Lykke.Service.Assets.Contract/Events/AssetUpdatedEvent.cs (hunk 1 above)
Comment (mmusin) [reply contract/Lykke.Service.Assets.Contract/Events/AssetUpdatedEvent.cs]: discussed

File: src/Lykke.Service.Assets/Services/AssetsHandler.cs [hunk 3]
```diff
@@ -45,7 +45,7 @@ public async Task<CommandHandlingResult> Handle(Commands.UpdateAssetCommand comm
 
             _chaosKitty.Meow("repository unavailable");
 
-            eventPublisher.PublishEvent(new AssetUpdatedEvent { Asset = command.Asset });
```
[the remark this answers is not in the document]
Comment (mmusin) [reply src/Lykke.Service.Assets/Services/AssetsHandler.cs]: + limitations

File: src/Lykke.Service.Assets/Services/AssetsHandler.cs [hunk 4]
```diff
@@ -34,7 +34,7 @@ public async Task<CommandHandlingResult> Handle(Commands.CreateAssetCommand comm
 
             _chaosKitty.Meow("repository unavailable");
 
-            eventPublisher.PublishEvent(new AssetCreatedEvent { Asset = command.Asset });
```
[the remark this answers is not in the document]
Comment (mmusin) [reply src/Lykke.Service.Assets/Services/AssetsHandler.cs]: limitations

**Pull request merged**